octobot/octobot.py

```diff
@@ -67,6 +68,9 @@ def __init__(self, config, ignore_config=False, reset_trading_history=False):
         # octobot_api to request the current instance
         self.octobot_api = OctoBotAPI(self)
 
+        # octobot channel global consumer
+        self.global_consumer = OctoBotChannelGlobalConsumer(self)
```

Comment:
should we nave this with octobot, something like ```octobot_global_consumer``` ?

Reply:
That was the original name but I renamed it because we are in OctoBot class. Do you prefer `octobot_global_consumer` ?

Reply:
Hum that's right actually !